src/helpers/send-bulk-message-helper.js

```diff
@@ -0,0 +1,66 @@
+export const isPotentialMobileNumber = (number) => {
```

Comment:
I'd probably name it as 'isNumberValid()` or something similar for readability

Reply:
changed to isValidMobileNumber format because were only checking the formatting here and not if the number exists.